SplunkRumCrashReporting/SplunkRumCrashReportingTests/DeviceStatsTests.swift

```diff
@@ -0,0 +1,35 @@
+//
+/*
+Copyright 2021 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@testable import SplunkRumCrashReporting
+import Foundation
+import XCTest
+
+class DeviceStatsTests: XCTestCase {
+    func testBattery() throws {
+        let batteryLevel = DeviceStats.batteryLevel
+        XCTAssertTrue(!batteryLevel.isEmpty)
+    }
+    func testFreeDiskSpace() throws {
+        let diskSpace = DeviceStats.freeDiskSpace
+        XCTAssertTrue(!diskSpace.isEmpty)
+    }
+    func testFreeMemory() throws {
+        let freeMemory = DeviceStats.freeMemory
+        XCTAssertTrue(!freeMemory.isEmpty)
+    }
+}
```

Comment:
The other testing element to consider is adding a second saved crash report with this new data to `CrashTests`.

Reply:
👍 I added another plcrash file and a second test for this rev.